wowchemy/layouts/partials/site_footer.html

```diff
@@ -1,4 +1,26 @@
 <footer class="site-footer">
+  {{/* Display any social links that the superuser chose to display in the footer. */}}
+  {{ $display_icons := false }}
+  {{ range where (where (where site.Pages "Section" "authors") ".Params.superuser" true) ".Params.social" "!=" nil }}
+    {{ range where .Params.social ".display.footer" true }}
+      {{ if not $display_icons }}
+        <div class="d-flex justify-content-center">
+          <ul class="network-icon" aria-hidden="true">
+      {{ end }}
+      {{ $display_icons = true }}
+      {{ $social_link := partial "functions/parse_social_link" . }}
```

Comment:
This should be calling [this function](https://github.com/wowchemy/wowchemy-hugo-modules/blob/master/wowchemy/layouts/partials/functions/get_social_link.html) right?

Reply:
Yes:)